crates/cargo-bsp/src/cargo_communication/cargo_types/cargo_command.rs

```diff
@@ -33,7 +50,16 @@ impl CreateCommand for RunParams {
         self.origin_id.clone()
     }
 
-    fn create_command(&self, root: PathBuf) -> Command {
+    fn create_unit_graph_command(&self, root: PathBuf) -> Command {
+        // TODO add appropriate build target to arguments
+        let mut cmd = Command::new(toolchain::cargo());
+        cmd.current_dir(root)
+            .args(["+nightly", "run", "--unit-graph", "-Z", "unstable-options"])
```

Comment:
sporo tych customowych argumentów się powtarza, może warto je wydzielić do jakichś stałych żeby pozbyć się magicznych napisów i uodpornić na potencjalne literówki

Reply:
Podobnie jak wyżej, jak Patryk skończy komendy to można to powydzielać

Reply:
no okay